crates/sui-types/src/object.rs

```diff
@@ -294,7 +294,7 @@ impl Owner {
         self == &Owner::Immutable
     }
 
-    pub fn is_owned(&self) -> bool {
+    pub fn is_owned_or_quasi_shared(&self) -> bool {
```

Comment:
What is "quasi_shared" here? It looks like this function only returns true if the object is owned by another object or by an address, so I am not sure where sharing comes in here...

As side note, this is `Impl` for the `Owner` enum but this enum does not define the owner a such but rather a type of ownership, so arguably `Ownership` would be a more suitable name.

Reply:
> It does give the owner, Owner::Address and Owner::Object both specify the actual address of the owner.

Indeed, but the enum also contains `Shared` and `Immutable` variants which do not define the owner as such but rather ownership type. I do understand that this is quite subjective, though.

Reply:
Agreed the wording is a bit wonky (and obviously subjective). I'd read that the same as if we did `owner: Option<Address>`, where this is sort of the negative case "I don't have an owner"